docs/user-guide/data-structures/data-structures.ipynb

```diff
@@ -54,11 +54,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "See [Creating Arrays and Datasets](./creating-arrays-and-datasets.rst) for an overview of the different methods for creating variables.\n",
+    "\n",
     "<div class=\"alert alert-info\">\n",
     "\n",
     "**Note:**\n",
     "\n",
-    "Internally scipp is not using numpy, so the above makes a *copy* of the numpy array of values into an internal buffer.\n",
+    "Internally scipp does use numpy, so the above makes a *copy* of the numpy array of values into an internal buffer.\n",
```

Comment:
No?

Reply:
Typo. Only meant to change away from a participle and not drop the 'not'.